### DISCUSSION TITLE
My custom domain is showing blank

### DISCUSSION THREAD

**Discussion by hainhtat** [community/community#23708] [Other Feature Feedback, Questions, & Ideas]
<p>My github pages is not showing index.html content. This is my website <a href="https://heinhtetaung.me" rel="noopener nofollow ugc">heinhtetaung.me</a>. This is my repository <a href="https://github.com/hainhtat/hainhtat.github.io/tree/gh-pages" rel="noopener nofollow ugc">https://github.com/hainhtat/hainhtat.github.io/tree/gh-pages</a>. When I visit the site, there’s nothing.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/8/c83764a72ce024882a67f212be0d8704843fce66.png" data-download-href="/uploads/short-url/szcaKDh2vEzY6pVAJl4SmwvXSoC.png?dl=1" title="Screen Shot 2020-10-21 at 02.38.00" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47028268/181078847-b7d82851-7369-4e36-b33d-d19c2f2bd1b8.png" alt="Screen Shot 2020-10-21 at 02.38.00" data-base62-sha1="szcaKDh2vEzY6pVAJl4SmwvXSoC" width="690" height="431" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/8/c83764a72ce024882a67f212be0d8704843fce66_2_690x431.png, https://aws1.discourse-cdn.com/github/optimized/3X/c/8/c83764a72ce024882a67f212be0d8704843fce66_2_1035x646.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/c/8/c83764a72ce024882a67f212be0d8704843fce66_2_1380x862.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/8/c83764a72ce024882a67f212be0d8704843fce66_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-10-21 at 02.38.00</span><span class="informations">2880×1800 140 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/1/61b428505c08361ae9c8734f9afe4bc81a490608.png" data-download-href="/uploads/short-url/dWkiC93VQAs3LedFDWjReZmYHiw.png?dl=1" title="Screen Shot 2020-10-21 at 02.38.07" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47028268/181078848-b540f6a1-8aab-4b4f-9d6e-87660bdd78d9.png" alt="Screen Shot 2020-10-21 at 02.38.07" data-base62-sha1="dWkiC93VQAs3LedFDWjReZmYHiw" width="690" height="431" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/1/61b428505c08361ae9c8734f9afe4bc81a490608_2_690x431.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/1/61b428505c08361ae9c8734f9afe4bc81a490608_2_1035x646.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/6/1/61b428505c08361ae9c8734f9afe4bc81a490608_2_1380x862.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/1/61b428505c08361ae9c8734f9afe4bc81a490608_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-10-21 at 02.38.07</span><span class="informations">2880×1800 525 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/0/109e3bd56f3e1d01110518da854e494757e0c325.png" data-download-href="/uploads/short-url/2n0EwTUBL6z1yhz5KWKTSBq2JPD.png?dl=1" title="Screen Shot 2020-10-21 at 02.38.15" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47028268/181078850-37e1b2d7-dbaf-4e94-9697-62e619ea0f41.png" alt="Screen Shot 2020-10-21 at 02.38.15" data-base62-sha1="2n0EwTUBL6z1yhz5KWKTSBq2JPD" width="690" height="431" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/0/109e3bd56f3e1d01110518da854e494757e0c325_2_690x431.png, https://aws1.discourse-cdn.com/github/optimized/3X/1/0/109e3bd56f3e1d01110518da854e494757e0c325_2_1035x646.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/1/0/109e3bd56f3e1d01110518da854e494757e0c325_2_1380x862.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/0/109e3bd56f3e1d01110518da854e494757e0c325_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-10-21 at 02.38.15</span><span class="informations">2880×1800 529 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by Valenchak** [comment, marked as answer]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/7/e765d0d145fa6666c7e365fbc0740a99f5639668.png" data-download-href="/uploads/short-url/x12qXkKmuPMTLysc696tVY8ceMM.png?dl=1" title="Снимок" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/840275/181078851-344fc671-3841-48a8-a4f7-b017940babc9.png" alt="Снимок" data-base62-sha1="x12qXkKmuPMTLysc696tVY8ceMM" width="690" height="308" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e765d0d145fa6666c7e365fbc0740a99f5639668_2_690x308.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e765d0d145fa6666c7e365fbc0740a99f5639668_2_1035x462.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/e/7/e765d0d145fa6666c7e365fbc0740a99f5639668.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e765d0d145fa6666c7e365fbc0740a99f5639668_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Снимок</span><span class="informations">1350×603 81.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
<a href="https://heinhtetaung.me/" rel="noopener nofollow ugc">Normally the resource opens</a>

**Comment by hainhtat** [comment]
<p>I’m still having the issue :”( </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/b/0ba4bece1e737b511762258951c0144a777b3401.jpeg" data-download-href="/uploads/short-url/1F0dtLNWNnjvffk1lOExd3qGNvb.jpeg?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47028268/181078853-730cf8ce-4fd1-4843-a9fb-66b68b659533.jpeg" alt="image" data-base62-sha1="1F0dtLNWNnjvffk1lOExd3qGNvb" width="230" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/b/0ba4bece1e737b511762258951c0144a777b3401_2_230x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/0/b/0ba4bece1e737b511762258951c0144a777b3401_2_345x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/0/b/0ba4bece1e737b511762258951c0144a777b3401_2_460x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/b/0ba4bece1e737b511762258951c0144a777b3401_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1125×2436 114 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
